aspnet/mvc/views/forms/sample/final/Views/DemoReg/RegRoute.cshtml

```diff
@@ -0,0 +1,8 @@
+﻿@model FormsTH.ViewModels.RegisterViewModel
+@addTagHelper "*, Microsoft.AspNet.Mvc.TagHelpers"
+
+<form asp-route="register" method="post">
+    Email:  <input id="Email" name="Email" /> <br>
+    Password: <input type="password" id="Password" name="Password" /><br>
+    <button type="submit">Register</button>
+</form>
```

Comment:
Recommend moving all files related to bare use of HTML into the startup demo. Further, recommend merging the two samples to reduce the file count e.g. `RegistrationController` and `RegistrationWithoutTagHelpersController`.